### DISCUSSION TITLE
How to zoom out or fitview to show 1,000s of nodes in the viewport?

### DISCUSSION THREAD

**Discussion by mjgleason3** [xyflow/xyflow#2849] [Q&A] [answered]
It looks like zoom out is restricted to 0.5, but for views with many nodes (1,000s) it can be helpful to zoom out and visualize the entire tree, even if you can't read the individual nodes. How can I enable fitview to update the zoom to go beyond the 0.5 limit?

**Comment by bcakmakoglu** [comment, marked as answer]
You can pass minZoom and maxZoom as props to the ReactFlow component, these values would then be used by `fitView`.
You can also pass a minZoom/maxZoom option to `fitView` (but if it doesn't match with the prop min/max you'd have a jump as soon as you start zooming, to match the min/max of the React Flow component).

**Comment by mahdizaabi** [comment]
@mjgleason3 did you manged to set default zoom to the view ???

**Reply by moklick** [reply]
Are you looking for the `defaultViewport` prop?

**Reply by mahdizaabi** [reply]
Yes, exactly! That is what I am looking for!
